imagelib/search.php

```diff
@@ -97,8 +97,8 @@
 			</ul>
 			<form name="imagesearchform" id="imagesearchform" action="search.php" method="post">
 				<div id="criteriadiv">
-					<div style="clear:both;height:50px">
-						<div style="float:left;margin-top:3px">
+					<div class="flex-form">
+						<div style="margin-top: 1.5px">
```

Comment:
Ideally, any inline styling that we have to change should become classes in a .css file. If the change applies to multiple files, it should go in css/v202209/symbiota/main.css. Otherwise, it should go in a .css file specific to this .php file. If one doesn't exist, it should be created.

In this case, with such a minor change and with other things more urgent, I think it's ok to ignore this rule.

Additionally, it might be nice to start migrating away from using px and use rem instead, because they are more 508-compliant; if users have to resize their screens, rem behaves better (but not always). It's worth testing different screen sizes and different levels of "zoom" with your browser as you find a good value for rem.